### PR TITLE
feat: better error page to display exceptions, update to FluentUI

### PR DESCRIPTION
**Pull request by JW-CH**

### CODE REVIEW

File: ImmichFrame/Views/SettingsView.axaml [hunk 1]
```diff
@@ -3,180 +3,181 @@
...
+		<Style Selector="TextBox">
+			<Setter Property="Margin" Value="10,3,10,10"/>
+			<Setter Property="VerticalAlignment" Value="Center"/>
+		</Style>
+		<Style Selector="CheckBox">
+			<Setter Property="Margin" Value="10,0,10,10"/>
+			<Setter Property="VerticalAlignment" Value="Center"/>
+		</Style>
+		<Style Selector="NumericUpDown">
+			<Setter Property="Margin" Value="10,3,10,10"/>
```
Comment (3rob3) [inline ImmichFrame/Views/SettingsView.axaml:27]: I'm not sure why, but the NumericUpDown's are not properly aligned anymore. This seems to fix it:
<Setter Property="Margin" Value="0,3"/>

File: ImmichFrame/Views/ErrorView.axaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+							Command="{Binding SettingsCommand}"/>
+						<Button
+							Content="Continue"
+							Command="{Binding ContinueCommand}"/>
+					</StackPanel>
+				</StackPanel>
+			</Border>
+		</Grid>
+	</DockPanel>
+</UserControl>
```
Comment (3rob3) [inline ImmichFrame/Views/ErrorView.axaml:46]: Love it on Desktop. Unfortunately it is not working on Android (ErrorView isn't shown at all).

Comment (JW-CH) [reply ImmichFrame/Views/ErrorView.axaml:46]: The ErrorView was working, just not showing on startup (When the settings were not correct). If you for example have a wrong ApiKey, the page will show. I made it more responsive, see my Image:
image

File: ImmichFrame/Views/SettingsView.axaml (hunk 1 above)
Comment (JW-CH) [reply ImmichFrame/Views/SettingsView.axaml:27]: I'm still not happy with the settings page. I will rework it in another PR.

File: ImmichFrame/Views/ErrorView.axaml (hunk 2 above)
Comment (3rob3) [reply ImmichFrame/Views/ErrorView.axaml:50]: Ah, ok. I should have tried creating a different type of error.

**Pull request merged**